### DISCUSSION TITLE
ABB charger working

### DISCUSSION THREAD

**Discussion by eogmau** [lbbrhzn/ocpp#168] [General]
Have tested the ABB Terra AC model W7-G5-R-0 and working, can be added to the list. The charger.Voltage is reporting 76.6V, not sure why. 
![IMG_20210921_192630](https://user-images.githubusercontent.com/21968661/134234684-755db8ff-6725-4d57-ab6a-2a99fae1c75a.jpg)

**Comment by lbbrhzn** [comment]
Looks like your charger is reporting only a single phase voltage, while OCPP is assuming three phases, so it is averaging (229.9 + 0 + 0) / 3 = 76.6 V. Created issue #169

**Comment by eogmau** [comment]
Another question for this charger.
The ABB charger has the set maxim Current slider greyed out, does it mean that the OCPP implementation on the charger doesn't support it?
Is there a way to check all the OCPP parameters supported on the charger apart from official documentation which doesn't exist.

**Reply by drc38** [reply]
From your logs the unit does not support the SmartCharging profile hence the greyed out slider. You can read up on the OCPP documentation what features each profile supports. A shorthand for what is supported is shown in the `<charger>.Features` sensor.

```
18:35:02 INFO (MainThread) [ocpp] ABBTerraAC: receive message [3, "81f1b569-2735-4dd8-a7b7-6c71ed41dab8", {"configurationKey": [{"key": "SupportedFeatureProfiles", "readonly": true, "value": "Core,FirmwareManagement,LocalAuthListManagement"}]
```

**Reply by eogmau** [reply]
Thanks for confirming.

**Comment by a42wg** [comment]
Hi, could someone share the OCPP server url to put into the charger from Terra config.
ABB are not very keen about giving access to the portal, but they have promised to add a configuration for me. So I may only have one shoot at this.
I found a screen shot for another charger, where the url was pointing to: ws://homeassistant.local:9000, would that work for the Terra AC?

reg Anders

**Reply by eogmau** [reply]
Hi, see a screenshot of my Terra Config OCPP configuration. You will need to use the ws://home-assistantIP:9000

**Comment by eogmau** [comment]
![IMG_20211130_184204](https://user-images.githubusercontent.com/21968661/144108289-6b359160-9f16-4320-9e4c-6b88dd24b4f6.jpg)

**Reply by a42wg** [reply]
Thank you, much appreciated

reg Anders
